styleguide/_themes/derek/scss/patterns/grid/ias.scss

```diff
@@ -1,17 +1,6 @@
 .ias-container {
-  @include container-fixed;
-  max-width: 960px;
+  @include make-container;
   padding: $standard-padding;
-
-  @media (min-width: $screen-sm-min) {
-    width: $container-sm;
-  }
-  @media (min-width: $screen-md-min) {
-    width: $container-md;
-  }
-  @media (min-width: $screen-lg-min) {
-    width: $container-lg;
-  }
```

Comment:
@amberRrucker I'm creating the `make-container` mixin instead, of having to use 11 lines of code to simulate that behavior every single time.  I've just modified the places that use the old behavior for simplicity.